iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingView.java

```diff
@@ -37,17 +37,116 @@
     /** To listen events from merging network */
     private final NetworkListener listener = new MergingNetworkListener();
 
-    private final NetworkBusBreakerViewAdapter busBreakerView;
+    static class BusBreakerViewAdapter implements Network.BusBreakerView {
```

Comment:
Why this class is private package? Private could be enough?
It seems it's required by unit tests. It's usually a bad idea to make the model weaker just for unit test

Reply:
My mistake, i forgot to put private after to move BusBreakerViewAdapter class into MergingView class